File: Java/src/Faccat/Exercicio5a.java
package Faccat;

import java.util.Scanner;

public class Exercicio5a {
    public static void main(String[] args) {
        int numero1, sucessor;
        Scanner scanner = new Scanner(System.in);

        System.out.print("Digite um número: ");
        numero1 = scanner.nextInt();

        sucessor = numero1 + 1;

        System.out.println("O antecessor do número digitado é: " + sucessor);

        scanner.close();
    }
}

File: JavaScript/Faccat/Exercicio5a.js
const readline = require('readline');

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

rl.question('Digite um número: ', function(numero1) {
  const sucess = parseInt(numero1) + 1;
  console.log('O antecessor do número digitado é: ' + sucess);

  rl.close();
});
